lib/bokken/accounts/organizer.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Bokken.Accounts.Organizer do
+  @moduledoc """
+  An organizer is a person who arranges the details of an event.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Bokken.Accounts.{Mentor, User}
+
+  @required_fields [:champion, :mentor_id, :user_id]
+  @optional_fields []
```

Comment:
In theory, organizers don't need to be mentors. For that reason, we could make `:mentor_id` a optional field.